example/ios/Podfile

```diff
@@ -1,4 +1,24 @@
 platform :ios, '9.0'
+use_frameworks!
+# use_modular_headers!
+
+dynamic_frameworks = []
+
+# make all the other frameworks into static frameworks by overriding the static_framework? function to return true
```

Comment:
Not sure it's a good idea to make the example uses `use_frameworks!` be default. It could scare beginners who won't have this issue anyway.

Reply:
ok, but if the beginner will copy this to his podfile - it will break nothing.

Reply:
Of course not. But the users who don't need it don't have to think this is mandatory. Commenting the whole + adding a similar comment as in the `README` could be a great solution.

Reply:
ok. done